ads/google/a4a/utils.js

```diff
@@ -331,3 +335,46 @@ export function additionalDimensions(win, viewportSize) {
           innerWidth,
           innerHeight].join();
 };
+
+/**
+ * Creates amp-analytics element within a4a element using urls specified
+ * with amp-ad closest selector and min 50% visible for 1 sec.
+ * @param {!../../../extensions/amp-a4a/0.1/amp-a4a.AmpA4A} a4a
```

Comment:
Nit: Clarify that it's the a4a _class_ and not the _element_.  (Should be obvious from type, but defensiveness is good...)